.gitignore

```diff
@@ -0,0 +1,11 @@
+venv
```

Comment:
[consider] Would be good to remove irrelevant entries here (e.g. `.vagrant`) and entries that should really be in local `.git/info/exclude` instead (e.g. `.idea`).

Reply:
[386e099](https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/3/commits/386e0991a7fa09bfe9edec6e3270c2c6382e3925)

Wasn't totally sure what was irrelevent and what wasn't, took a stab.

Reply:
[bd65223](https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/3/commits/bd65223d555fac9ab933a92c2e7d7b8d738d9d55)

Removed `.pybuild` and `venv`